moskito-frontend/src/Pages/Bookmarks.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types'
+import CampsiteList from '../components/Campsitelist/CampsiteList'
+import Header from '../components/Header/Header'
+import Navigation from '../components/Navigation/Navigation'
+import { useEffect } from 'react'
+import useCampsites from '../hooks/useCampsites'
+
+
+Campsites.propTypes = {
+    headline: PropTypes.string.isRequired
+}
+
+export default function Campsites({headline}) {
+    const { bookmarks, getBookmarks } = useCampsites()
+
+    useEffect(() => {
+        getBookmarks()        
+    }, [])
+    
+    return (
+        <>
+            <Header children={headline}/>
+            <CampsiteList campsites={bookmarks}/>
+            <Navigation/>
+        </>
+    )   
+}
```

Comment:
Looks like prettier is not working properly, usually it would add another line here